mobly/test_runner.py

```diff
@@ -35,7 +35,7 @@
 from mobly import utils
 
 
-def main():
+def main(argv=sys.argv[1:]):
```

Comment:
You should never provide mutable default arguments in python:
http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments